addons/web/static/src/views/list/list_renderer.js

```diff
@@ -1775,7 +1775,9 @@ export class ListRenderer extends Component {
         const cells = [...element.querySelectorAll("td")];
         for (const [index, header] of Object.entries(headers)) {
             const style = getComputedStyle(header);
-            cells[index].style.width = style.width;
+            if(cells[index]){
+                cells[index].style.width = style.width;
+            }
```

Comment:
Seems unnecessary to call `getComputedStyle` as well then. How about this?

```suggestion
            if (!cells[index]) {
                continue;
            }
            const style = getComputedStyle(header);
            cells[index].style.width = style.width;
```

Reply:
maybe just call everything inside the if statement then? and if we still would want to have them outside of statement, wouldn't "break" be smarter in this case rather then "continue"? if index with specific value doesn't exist, then it'd be better to just break out of the loop and not continue iterating over empty indexes no?

Reply:
1. Definitely!
2. Summarizing what we talked about, there is also the colspan of the section cells (and reordering of columns) to take into consideration, early break is likely not applicable in this case.

Reply:
So apparently https://github.com/odoo/odoo/pull/110155/files Here h4818 came up with the solution for our colspan issue by. I tested and reusing code snippet fixes our colspan issue pretty well in our case, so I think I'll use the similar logic here too.

Reply:
Nice, but then don't because we should soon all benefit from it (check Discord ;-) ).

Reply:
Cool, I'll close this pull request in that case :D